man/systemd.net-naming-scheme.xml

```diff
@@ -457,6 +466,18 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><constant>v254</constant></term>
+
+          <listitem><para>Naming was changed for SR-IOV VF representors.</para>
+
+          <para>SR-IOV VF representors are named based on the name of the PF interface, with a suffix of
+          <literal>r<replaceable>port</replaceable></literal>, where <replaceable>port</replaceable> is the
+          number of the virtual device associated with the representor.
```

Comment:
No need to repeat the docs above (which might change anyway). Maybe just say "The  <literal>r<replaceable>port</replaceable></literal> suffix was added to differentiate SR-IOV VF representors attached to a single PF interface". (IIUC)

Reply:
Agree, will fix